GitHub/GitHub_Create_Pull_Request.ipynb

```diff
@@ -0,0 +1,276 @@
+    "- [GitHub Documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "314a3e2b-68cf-4ed8-af3e-83fc52a42590",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "438e3250-f55d-4f66-8233-d18832c55369",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55b6723a-6d1d-42c7-a1bd-764d9c3b4333",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import github"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08549840-69f5-4a90-aee7-2d9ce922235c",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup Variables\n",
+    "- `token`: [GitHub token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)\n",
+    "- `repo`: repository name\n",
+    "- `owner`: repository owner\n",
+    "- `title`: pull request title\n",
+    "- `body`: pull request body\n",
+    "- `head`: branch name where your changes are implemented\n",
+    "- `base`: branch name where your changes will be merged into\n",
+    "- `assignees`: list of assignees\n",
+    "- `issue`: issue number"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d89c4f3f-21a7-49b6-9112-81267d432820",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "token = \"YOUR_TOKEN\"\n",
+    "repo = \"YOUR_REPO\"\n",
+    "owner = \"YOUR_OWNER\"\n",
+    "title = \"YOUR_TITLE\"\n",
+    "body = \"YOUR_BODY\"\n",
+    "head = \"YOUR_HEAD\"\n",
+    "base = \"YOUR_BASE\"\n",
+    "assignees = [\"ASSIGNEE_1\", \"ASSIGNEE_2\"]\n",
+    "issue = 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35112173-4135-4a21-a86f-69cd65ae2694",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3a9db93-3a28-4327-ad32-c008f98ec75d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Create Pull Request"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61f41916-cfb3-4bf6-b29d-2bb9b9bb3ffd",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "Create a pull request with assignee and link to issue using pygithub library."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ee8eb20f-dfe2-4f09-b8c8-fac86d6bd6d3",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Create a Github instance\n",
+    "gh = github.Github(token)\n",
```

Comment:
<img width="756" alt="Screenshot 2023-02-18 at 12 40 27 AM" src="https://user-images.githubusercontent.com/81156510/219765379-6faa6852-6248-400b-9bef-e34716f1aba1.png">

- Getting this error while running after declaring the environment variables.
- Could be a possible [fix](https://pygithub.readthedocs.io/en/latest/examples/PullRequest.html).